packages/common/src/tools/promiseAllPlus.ts

```diff
@@ -59,6 +59,7 @@ export async function promiseAllPlus<T>(
   await taskQueue.waitTilEmpty()
 
   if (permanentError) {
+    // eslint-disable-next-line @typescript-eslint/no-throw-literal
```

Comment:
Possibly a stupid rule that we could disable for the whole codebase.

Reply:
@krzkaczor Agree, disabled